ansible/roles_ocp_workloads/ocp4_workload_quarkus_super_heroes_demo/tasks/post_workload.yml

```diff
@@ -31,9 +31,14 @@
         - Villains service data UI: http://rest-villains-{{ t_project_name }}.{{ route_subdomain }}
         - Apicurio Schema Registry: http://apicurio-{{ t_project_name }}.{{ route_subdomain }}
         - Prometheus: http://prometheus-operated-{{ t_project_name }}.{{ route_subdomain }}
-
   loop: "{{ ocp4_workload_quarkus_super_heroes_demo_java_versions | product(ocp4_workload_quarkus_super_heroes_demo_app_kinds) | list }}"
 
+- name: output cicd info
```

Comment:
If you do msg make sure you also do data to save the data to the data store.

Reply:
Will do.